roles/certificate-acl/templates/certificate_acl_redis_statefulset.yaml.j2

```diff
@@ -0,0 +1,35 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  labels:
+    name: certificate-acl-redis
+    app: dojot
+  name: certificate-acl-redis
+  namespace: {{ dojot_namespace }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: dojot
+      name: certificate-acl-redis
+  template:
+    metadata:
+      labels:
+        name: certificate-acl-redis
+        app: dojot
+    spec:
+      restartPolicy: Always
+      containers:
+      - image: dojot/redis:6.0.4-alpine3.11
+        name: certificate-acl-redis
+{% if dojot_enable_node_affinity %}
+      affinity:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+            - matchExpressions:
+              - key: dojot.components/dojot
```

Comment:
I think you should change to `vernemq` label like this:

`dojot.components/vernemq`